app/routes/events/components/EventDetail/index.tsx

```diff
@@ -395,29 +398,34 @@ export default class EventDetail extends Component<Props, State> {
                 <Icon name="time-outline" className={styles.infoIcon} />
                 <FromToTime from={event.startTime} to={event.endTime} />
               </div>
-              <div className={styles.iconWithText}>
-                <Icon name="location-outline" className={styles.infoIcon} />
-                {event.location}
-              </div>
-              {event.isPriced && (
+              <div className={styles.iconWithTextMazeMap}>
                 <div className={styles.iconWithText}>
-                  <Icon name="cash-outline" className={styles.infoIcon} />
-                  {event.priceMember / 100},-
+                  <Icon name="location-outline" className={styles.infoIcon} />
+                  {event.location}
                 </div>
-              )}
-              {event.mazemapPoi && (
-                <>
-                  <div
-                    className={styles.simulateLink}
+                {event.mazemapPoi && (
+                  <Button
+                    className={styles.mapButton}
                     onClick={() =>
                       this.setState({
                         mapIsOpen: !this.state.mapIsOpen,
                       })
                     }
                   >
-                    {' '}
+                    <CircularPicture size={25} alt="" src={mazemapLogo} />
```

Comment:
Just use the `<Image />` component. This logo is not circular.

Reply:
Also, never do `alt=""`

Reply:
done